site-scrapers/Walgreens/index.js

```diff
@@ -12,7 +12,12 @@ module.exports = async function GetAvailableAppointments(browser) {
         site.locations,
         site.website
     );
-    return results;
+    return {
+        parentLocationName: "Walgreens",
+        isChain: true,
+        timestamp: moment().format(),
+        individualLocationData: await results,
+    };
```

Comment:
it's dumb, the answer is it wasn't broken because it wasn't actually an asynchronous function but for some reason Fauna was picking up on it being a Promise instead of an array. In any case, I actually don't need the async function at all so I rewrote it a little.